spacy_llm/pipeline/llm.py

```diff
@@ -21,6 +21,20 @@
 logger = logging.getLogger("spacy_llm")
 logger.addHandler(logging.NullHandler())
 
+DEFAULT_MODEL_CONFIG = {
+    "@llm_models": "spacy.GPT-3-5.v1",
```

Comment:
```suggestion
    "@llm_models": "spacy.GPT-3-5.v2",
```

Reply:
Like I said - this requires https://github.com/explosion/spacy-llm/pull/236 to be merged first, or this PR's CI will fail